### PR TITLE
Fix CHANGELOG formatting for PyPI upload & bump to 3.2.1

### PR DESCRIPTION
**Pull request by rbm**

### CODE REVIEW

**Comment by rogerhu** [comment]
lgtm

**Comment by asherf** [comment]
Lgtm 👋👋👋

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 
-3.2.0
-------------------
+3.2.1
+--------------------
+    - Fix rST formatting in this file to pass PyPI rendering check
+
+3.2.0 (Not released)
+--------------------
     - Support Django 2.3
```
Comment (shurph) [inline CHANGELOG.rst]: It's about Django **2.2** support, not 2.3.

`DJANGO_VERSION_CEILING` is used as:
>  Django **<** `DJANGO_VERSION_CEILING`

I was confused by meaning of `DJANGO_VERSION_CEILING`, too :-)

Comment (rbm) [reply CHANGELOG.rst]: Thanks for the catch!

**Pull request merged**